### DISCUSSION TITLE
How can I right align column header and cell?

### DISCUSSION THREAD

**Discussion by slakbal** [rappasoft/laravel-livewire-tables#347] [Q&A]
I have a "not sortable" amount column and it best viewed with a right alignment. I have succeeded in right aligning the specific cell contents. Any tips on how I can right align the header also? 

I've looked at the code in heading.blade.php. Would be awesome if there were also some hooks into this for each column and row similar like "setTableRowClass". But on columns that are not sortable the $column prop is not passed through.

```blade
@props([
    'column',
    'sortable' => null,
    'direction' => null,
    'text' => null,
])

<th
    {{ $attributes->merge(['class' => 'px-3 py-2 md:px-6 md:py-3 bg-gray-50'])->only('class') }}
>
    @unless ($sortable)
        <span class="block text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider">
            {{ $text ?? $slot }}
        </span>
    @else
        <button
            wire:click="sortBy('{{ $column }}', '{{ $text ?? $column }}')"
            {{ $attributes->except('class') }}
            class="flex items-center space-x-1 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider group focus:outline-none focus:underline"
        >
```

Any ideas on how I can solve this elegantly? Would have been awesome if one could drive column (and header) alignment straight from the columns definition and apply it to the row cell and the header cell. See below for some example variants.

```php
public function columns(): array
    {
        return [
            Column::make('Created', 'created_at'),
            Column::make('Description', 'description'),
            Column::make('Amount', 'amount')->alignRight(),
        ];
    }
```

Would love some feedback! 

Thanks

**Comment by rappasoft** [comment]
Try:

```php
Column::make('Amount', 'amount')->addClass('text-right'),
```

**Reply by slakbal** [reply]
hi @rappasoft thanks for the reply. I did use that and it works fine for the column, but not to the header. Hence me mentioning even on a "non sortable" column. The problem with the columns are that they are "auto" spacing as such the header is super misaligned e.g. on a decimal amount column which is typically right aligned.
